guide/private-key-management/external-signer.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: guide
+title: External signers
+description: What External signers are and their role in securing bitcoin.
+nav_order: 4
+parent: Private key management
+permalink: /guide/private-key-management/external-signers/
+image: https://bitcoin.design/assets/images/guide/private-key-management/external-signers/external-signing-device-preview.png
+redirect_from:
+ - /guide/onboarding/external-signing-device/
+main_classes: -no-top-padding
+---
+
+<!--
+
+Editor's notes
+
+Description of what an external signers are and what the scheme consists of.
+
+Illustration sources
+
+https://www.figma.com/community/file/888680264445459448
+https://www.figma.com/community/file/995256542920917246/BDG---Private-key-management-illustrations
+https://www.figma.com/file/qr4P17z6WSPADm6oW0cKw2/?node-id=261%3A1837
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/private-key-management/external-signers/external-signing-device.png"
+   retina = "/assets/images/guide/private-key-management/external-signers/external-signing-device@2x.png"
+   mobile = "/assets/images/guide/private-key-management/external-signers/external-signing-device-mobile.png"
+   mobileRetina = "/assets/images/guide/private-key-management/external-signers/external-signing-device-mobile@2x.png"
+   alt-text = "Examples of popular external signers."
+   width = 1600
+   height = 700
+   layout = "full-width"
+%}
+
+# External signers
+
+External signers use, store and manage private keys and other sensitive info, such as the users recovery phrase, externally from the users wallet application. The most common external signers used today are hardware wallets. Software only external signing applications also exist, but are less secure than their hardware counterparts. 
+
+A wallet application that supports external signers can remove the need for the private keys to be store locally by the wallet application. This decreases the risk of theft as the keys can be isolated from the internet and/or a potentially malicious device. 
+
+### How it works
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/private-key-management/external-signers/external-signer.jpg"
+   retina = "/assets/images/guide/private-key-management/external-signers/external-signer@2x.jpg.jpg"
```

Comment:
```suggestion
   retina = "/assets/images/guide/private-key-management/external-signers/external-signer@2x.jpg"
```

Reply:
The three icons don't quite match. Different line thicknesses and levels of detail.

Reply:
Will fix now :)